sklearn/decomposition/tests/test_sparse_pca.py

```diff
@@ -70,6 +71,10 @@ def test_fit_transform():
     spca_lasso.fit(Y)
     assert_array_almost_equal(spca_lasso.components_, spca_lars.components_)
 
+    # Test calling tranform with deprecated ridge_alpha parameter
```

Comment:
Seeing the note about testing times with SparsePCA I stuck the deprecation test in here. Lmk if you think it's better to break it out.